_papers/thesis-report/additional/extended-abstract.typ

```diff
@@ -351,6 +357,8 @@ These organization structures are then evaluated using queries that test five di
 + deleting a resource.
 ]
 
+// TODO: We definitely need to mention some results here in some way! E.g. via a figure or table. Feel free to shorten future work to make space for this.
```

Comment:
Okey! I'll see what I can do here. I'll probably end up showing the tables of the "create resource" and "move resource" choke points. Do you think that would suffice?

Reply:
Sure.